.travis.yml

```diff
@@ -23,11 +23,10 @@ before_install:
   - truncate -s 100G /tmp/disk.img
   - sudo zpool create -f zfspv-pool /tmp/disk.img
 install:
-  - if [ "$TRAVIS_BUILD_DIR" != "$GOPATH/src/github.com/openebs/zfs-localpv" ]; then
-    mkdir -p $GOPATH/src/github.com/openebs/;
-    mv $TRAVIS_BUILD_DIR $GOPATH/src/github.com/openebs;
-    cd $GOPATH/src/github.com/openebs/zfs-localpv;
-    fi
+  - if [ "$TRAVIS_BUILD_DIR" != "$HOME/openebs/zfs-localpv" ]; then
+    mkdir -p $HOME/openebs/;
+    mv $TRAVIS_BUILD_DIR $HOME/openebs;
+    cd $HOME/openebs/zfs-localpv;
```

Comment:
i think we don't need this anymore , we can remove this , travis handles the gopath correctly now.

Reply:
can I just remove it completely then?

Reply:
yes, please